kaggle_environments/envs/llm_20_questions/llm_20_questions.py

```diff
@@ -86,16 +86,21 @@ def answerer_agent(obs):
 
 agents = {GUESSER: guesser_agent, ANSWERER: answerer_agent}
 
+def limit_length(input, size):
+    return input[:size] if len(input) > size else input
```

Comment:
nit: I don't think you need this function, depends on type of `input` I guess but usually you can slice past the end and it'll work correctly.